spec.md

```diff
@@ -769,7 +769,7 @@ As discussed in Section 2.2, the IKM input to KeyGen MUST be infeasible to guess
    
 Secret keys MAY be generated using other methods; in this case they MUST be infeasible to guess and MUST be indistinguishable from uniformly random modulo r.
 
-BBS signatures are nondeterministic, meaning care must be taken against attacks arising from signing with bad randomness, for example, the nonce reuse attack on ECDSA [HDWH12]. It is recommended that the nonces used in signature proof generation are from a trusted source of randomness.
+BBS signatures are nondeterministic, meaning care must be taken against attacks arising from signing with bad randomness, for example, the nonce reuse attack on ECDSA [HDWH12]. It is recommended that the nonces used in signature proof generation are from a trusted source of randomness (see Nonce selection section below).
```

Comment:
I don't know what standard in cryptography is used to reference to good randomness. Maybe [this RFC](https://datatracker.ietf.org/doc/html/rfc4086) may be more appropriate to quote here?